supabase/migrations/20260828122104_org_support_channel.sql

```
@@ -0,0 +1,108 @@
+  END IF;
+
+  RETURN NEW;
+END;
+$$;
+
+ALTER FUNCTION public.guard_org_support_channel() OWNER TO postgres;
+REVOKE ALL ON FUNCTION public.guard_org_support_channel() FROM PUBLIC;
+GRANT EXECUTE ON FUNCTION public.guard_org_support_channel() TO service_role;
+
+COMMENT ON FUNCTION public.guard_org_support_channel() IS
+  'BEFORE INSERT OR UPDATE trigger on public.orgs (per row). Blocks client writes '
```

SQLFluff (4.3.0)

[error] Line is too long (83 > 80).

(LT05)